js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -637,3 +641,126 @@ impersonate=Impersonieren
+enableLdapv3Password=Erweiterten LDAPv3-Kennwortänderungsvorgang aktivieren
+enableLdapv3PasswordHelp=Wenn aktiv, wird die LDAPv3 Password Modify Extended Operation (RFC-3062) verwendet. Die erweiterte Kennwortänderung setzt normalerweise voraus, dass der LDAP-Benutzer bereits ein Kennwort im LDAP-Server hat. Wenn dies also mit "Sync Registrations" verwendet wird, kann es sinnvoll sein, auch "Hardcoded LDAP-Attribute mapper" mit zufällig generiertem Initialpasswort hinzuzufügen.
+validatePasswordPolicy=Kennwortrichtlinie validieren
+validatePasswordPolicyHelp=Legt fest, ob Keycloak das Passwort mit der Realm-Passwortrichtlinien validieren soll, bevor es aktualisiert wird. Für den Fall, dass das Passwort des Benutzers in LDAP gespeichert ist, werden einige Keycloak-Passwortrichtlinien nicht funktionieren (z.B. Not Recently Used, Expire Password, Hashing Iterations, Hashing Algorithm), da Keycloak keine direkte Kontrolle über die Passwortspeicherung hat. Es ist notwendig, Passwortrichtlinien auf der LDAP-Server-Ebene zu aktivieren, wenn Sie diese Passwortrichtlinien nutzen wollen.
+trustEmail=E-Mail vertrauen
+trustEmailHelp=Wenn diese Option aktiviert ist, wird die von diesem LDAP-Anbieter bereitgestellte E-Mail nicht überprüft, selbst wenn die Überprüfung für den Realm aktiviert ist.
+queryExtensions=Unterstützte Erweiterungen abfragen
+syncChangedUsers=Geänderte Benutzer synchronisieren
+syncAllUsers=Alle Benutzer synchronisieren
+removeImported=Importierte Benutzer entfernen
+deleteProvider=Anbieter löschen?
+searchForMapper=Suche nach Mapper
+addMapper=Mapper hinzufügen
+mapperType=Mapper Typ
+mapperTypeHelp=Wird verwendet, um ein einzelnes Attribut eines LDAP-Benutzers auf ein Attribut des UserModels in Keycloak DB abzubilden.
+userModelAttribute=Benutzermodell-Attribut
+custom=Benutzerdefiniertes Attribut...
+ldapAttribute=LDAP-Attribut
+ldapAttributeHelp=Name des zugeordneten Attributs im LDAP-Objekt, zum Beispiel "cn", "sn", "mail", "street", usw.
+readOnly=Nur lesen
+readOnlyHelp=Ein schreibgeschütztes Attribut wird aus LDAP in das UserModel importiert, aber es wird nicht zurück in LDAP gespeichert, wenn der Benutzer in Keycloak aktualisiert wird.
+alwaysReadValueFromLdap=Wert immer aus LDAP lesen
+alwaysReadValueFromLdapHelp=Wenn aktiv, dann wird beim Lesen immer das LDAP-Attribut anstatt der Wert aus der Keycloak DB verwendet.
+isMandatoryInLdap=Ist in LDAP erforderlich
+isMandatoryInLdapHelp=Wenn aktiv, ist das Attribut in LDAP erforderlich. Wenn kein Wert in der Keycloak-DB vorhanden ist, wird ein leerer Wert gesetzt, und an LDAP weitergegeben.
+attributeDefaultValue=Attribut-StandardwertClick to apply
+attributeDefaultValueHelp=Wenn es keinen Wert in Keycloak DB gibt und das Attribut in LDAP erforderlich ist, dann wird dieser Wert an LDAP weitergegeben.
```

Comment:
Keycloak-DB

Reply:
Replaced all occurences of `Keycloak DB` with `Keycloak-DB`.